### PR TITLE
(closes #233): VM status check before affinity group changing

### LINKED ISSUE #233: Affinity group change: "VM stop" message bug
Steps to reproduce:

1. Go to VM details of a running VM
2. Click "Affinity group change" button
3. Click "Stop" in a system message ("To change VM you need to stop VM. Are you sure you want to stop it?")

VM is stopped, the affinity group is changed, it is ok. 

4. Do not close sidebar, click "Affinity group change" button again straight away.

Expected result: Dialogue with existing groups opens directly, no system message.
Actual result: System message about VM stopping appears, though VM is already stopped.

### PR DESCRIPTION
**Pull request by zolotyx**
fix(vm): Check for VM state before changing affinity group

### CODE REVIEW

File: src/app/vm/vm-sidebar/vm-detail.component.ts [hunk 1]
```diff
@@ -151,10 +148,7 @@ export class VmDetailComponent implements OnChanges, OnInit {
   }
 
   public resetSshKey(): void {
-    this.askToStopVM(
-      'STOP_MACHINE_FOR_SSH',
-      () => this.showSshKeypairResetDialog()
-    );
+    this.askToStopVM('STOP_MACHINE_FOR_SSH').subscribe(() => this.showSshKeypairResetDialog());
```
Comment (andrewbents) [inline src/app/vm/vm-sidebar/vm-detail.component.ts]: add some loading indicator

File: src/app/vm/vm-list/vm-list.component.ts [hunk 2]
```diff
@@ -73,6 +73,19 @@ export class VmListComponent implements OnInit {
     this.subscribeToVmUpdates();
     this.subscribeToVmDestroyed();
     this.subscribeToVmCreationDialog();
+    this.asyncJobService.event.subscribe((job: AsyncJob<any>) => {
```
Comment (andrewbents) [inline src/app/vm/vm-list/vm-list.component.ts]: add filtering event by entity

File: src/app/vm/vm-sidebar/vm-detail.component.ts (hunk 1 above)
Comment (zolotyx) [inline src/app/vm/vm-sidebar/vm-detail.component.ts]: @andrewbents added indicator for AG and SSH Key while dialog is opened

File: src/app/vm/vm-list/vm-list.component.ts (hunk 2 above)
Comment (zolotyx) [inline src/app/vm/vm-list/vm-list.component.ts]: @andrewbents , filter added

**Pull request merged**